common/scylla_configure.py

```diff
@@ -50,14 +48,22 @@ class ScyllaAmiConfigurator:
         'start_scylla_on_first_boot': True,
     }
 
-    INSTANCE_METADATA_URL = "http://169.254.169.254/latest/"
     DISABLE_START_FILE_PATH = Path("/etc/scylla/ami_disabled")
```

Comment:
We should rename the file to something else, no? But let's also keep `ami_disabled` for backward compatibility.

Reply:
This name can't be change without changing it in code, this is the file that stop scylla service for starting.

It can be changed, but it need to be in sync with scylla-core.

Reply:
yeah, I didn't want to change the legacy code, that's why I kept this. We can do this in the future as an improvement.

Reply:
Please open a ticket for this cleanup so we don't forget.